services/getstream/GetstreamService.js

```diff
@@ -12,7 +12,7 @@ class GetstreamService {
     const query = {
       limit,
       reactions: {own: true, recent: true, counts: true},
-      ranking: GETSTREAM_RANKING_METHOD,
+      // ranking: GETSTREAM_RANKING_METHOD,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `ranking` parameter will cause the feed to default to chronological order. If this is intended, then it's fine. However, if you want to maintain the ability to switch between ranking methods, consider making the ranking method a parameter of the function with a default value of 'chronological'. This way, you can easily switch back to using a ranking method when needed.
```diff
- // ranking: GETSTREAM_RANKING_METHOD,
+ ranking: rankingMethod || 'chronological',
```
Please ensure that `rankingMethod` is passed as an argument to the function where this code resides.


<!-- This is an auto-generated comment by OSS CodeRabbit -->